azafea/event_processors/endless/activation/tests/integration/test_activation_v1.py

```diff
@@ -117,4 +117,4 @@ def test_activation_v1_invalid_image(self, capfd):
             assert dbsession.query(Activation).count() == 0
 
         capture = capfd.readouterr()
-        f"Invalid image id 'image': Did not match the expected format" in capture.err
+        assert "Invalid image id 'image': Did not match the expected format" in capture.err
```

Comment:
This change has exposed a pre-existing bug: this assertion fails. (Previously the `is` expression evaluated to `False` which was discarded.)

I suggest separating this patch ("Remove useless f-strings") to a separate PR, unless there's a strong reason to have it in this PR?

Reply:
> I suggest separating this patch ("Remove useless f-strings") to a separate PR, unless there's a strong reason to have it in this PR?

The only reason was that they prevent tests to pass. I don’t know why flake8 suddenly decided that it was bad.

If you want, I can create a PR fixing f-strings that can be merged quickly, and rebase this branch when it’s merged.

Reply:
I see, can I suggest 3 separate PRs?

1. The patch to replace redundant `f""` with `""` (but without adding the missing `assert`s)
2. The rest of this PR, built atop 1
3. A patch to add the missing `assert`s you've found and address the issues caught by adding them, also built atop 1

Reply:
#137 #138